dataworks-aws-data-egress.tf

```
@@ -0,0 +1,81 @@
+locals {
+  dataworks_aws_data_egress_pipeline_name = "dataworks_aws_data_egress"
```

you might want this to be the shorter form:
```suggestion
  dataworks_aws_data_egress_pipeline_name = "data_egress"
```

---

done